src/render/filters.rs

```diff
@@ -401,6 +401,15 @@ mod tests {
 
             assert_eq!(result, "a-b");
 
+            let engine = EngineData::empty();
+            let template_string = "{{ var|slugify }}".to_string();
+            let context = PyDict::new(py);
+            context.set_item("var", 1).unwrap();
```

Comment:
@LilyFoote It appears that even if we add an integer here this is still effectively treated as if this coming from the python arm. Do you have any thoughts around this?

Reply:
Yeah, `context` is basically a `HashMap<String, Py<PyAny>>`, so putting an integer in will convert to a Python `int`.

Instead, the way to do this is to chain multiple filters. I usually use `default` for this:

```
{{ var|default:1|slugify }}
```

The result of the `default` filter will be a `Content::Int(1)` if `var` is not defined.

Reply:
I see, that means that for the majority of cases this won't matter much. Though I guess the type of `HashMap` may change at some point to make this more relevant.